test/recipes/default/user_test.rb

```diff
@@ -0,0 +1,71 @@
+describe group('stanley') do
+  it { should exist }
+end
+
+describe user('stanley') do
+  it { should exist }
+  its('group') { should eq 'stanley' }
+  its('home') { should eq '/home/stanley' }
+end
+
+describe passwd.users(/stanley/) do
+  its('homes') { should eq ['/home/stanley'] }
+  its('shells') { should eq ['/bin/bash'] }
```

Comment:
Yeah, it's not a requirement for StackStorm. Looks like `bash` is unnecessary information here.